sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_helpers.cs

```diff
@@ -217,7 +217,7 @@ public virtual async Task<Uri> GetClientAccessUriAsync(DateTimeOffset expiresAt,
         }
 
         /// <summary>
-        /// Creates a URI with authentication token for clients.
+        /// Creates a URI with authentication token for clients.Creates a URI with authentication token for the clients. Specify clientEndpointType to MQTT for MQTT clients, others use the default one.
```

Comment:
? duplicate sentences? use this whole description for all the overloads : `Creates a URI with authentication token for the clients. Specify clientEndpointType to MQTT for MQTT clients, others use the default one.`